app/src/screen.c

```diff
@@ -283,9 +285,38 @@ event_watcher(void *data, SDL_Event *event) {
 
     if (event->type == SDL_WINDOWEVENT
             && event->window.event == SDL_WINDOWEVENT_RESIZED) {
-        // In practice, it seems to always be called from the same thread in
-        // that specific case. Anyway, it's just a workaround.
-        sc_screen_render(screen, true);
+        // idk
+        if (screen->is_virtual) { //((sz.width % 16) == 0 && (sz.height % 16) == 0) {
+            // FIXME: probably slow
+            struct sc_size sz = get_window_size(screen);
+            struct sc_control_msg msg;
+            msg.type = SC_CONTROL_MSG_TYPE_RESIZE_DISPLAY;
+            msg.resize_display.width = sz.width;
+            msg.resize_display.height = sz.height;
```

Comment:
When I resize my client window, the captured display remains at the initial resolution (which is fine, but I think this code was expected to resize on the device, right?). In the end, it should probably be an option.